### PR TITLE
Rename process to internalProcess

### PR DESCRIPTION
**Pull request by tbranyen**
This fixes GH-268. Webpack has issues with the process variable and by
renaming the internal usage we can ensure that it points to our
representation and not the external one.

### CODE REVIEW

File: packages/diffhtml-components/lib/util/process.js [hunk 1]
```diff
@@ -1,7 +0,0 @@
-import diff from './binding';
```
Comment (tbranyen) [inline packages/diffhtml-components/lib/util/process.js:1]: Turns out this file wasn't used.

**Comment by justjake** [comment]
Another day, another Webpack issue. But, I think your Rollup config is already explicitly trying to replace `process.env.NODE_ENV` statically at build time in a few places:

image

**Comment by tbranyen** [comment]
Good catch, thanks for the review!

File: packages/diffhtml/rollup.main.config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import babel from 'rollup-plugin-babel';
 import nodeResolve from 'rollup-plugin-node-resolve';
-import replace from 'rollup-plugin-replace';
+import replace from '@rollup/plugin-replace';
```
Comment (tbranyen) [inline packages/diffhtml/rollup.main.config.js:3]: The old rollup-plugin-replace did not work for some reason, upgrading seems to fix the issue.

**Pull request merged**